src/handlers/gestures/GestureDetector.tsx

```diff
@@ -35,6 +36,9 @@ import { ComposedGesture } from './gestureComposition';
 import { ActionType } from '../../ActionType';
 import { isFabric, tagMessage } from '../../utils';
 import { getShadowNodeFromRef } from '../../getShadowNodeFromRef';
+import { Platform } from 'react-native';
+import RNGestureHandlerModuleWeb from '../../RNGestureHandlerModule.web';
```

Comment:
I'd prefer to avoid importing like this. In general `.web.ts` files should never be imported manually, as it is expected that they are specific for web and therefore can bring a ton of web-only dependencies (e.g. hammer.js) with them. I think it'd be best to have this abstracted in `RNGestureHandlerModule` directly (then the module would have separate impl for mobile/web)

Reply:
This comes from the fact that the `RNGestureHandlerModule.attachGestureHandler()` has a different signature on web - it takes one more argument containing the ref with the callbacks. I don't see how it can be easily abstracted, other than adding an argument that will be ignored on native platforms.

I changed it in [07e620a](https://github.com/software-mansion/react-native-gesture-handler/pull/1934/commits/07e620ab2151c9d5a260dab0b24348dc42113658) to only import type (the same way it's already implemented in the `createHandler.tsx`). Is that enough or should I do it differently?